index.html

```diff
@@ -192,14 +320,78 @@ <h1 class="main-text">About Myself</h1>
             </ul>
         </div>
     </section>
+    <!--  Desktop About --> 
+    <section class="no-about">    
+        <div class="btl-rad about">
+            <div class="about about-desktop">
+                <br>
+                <h1 class="main-text">About <br> Myself</h1>
+                <p class="content">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me..</p>
+                <br>
+                <p id="about-link">LET'S CONNECT</p>
+                <br>
+                <div class="contact">
+                    <ul class="list">
+                        <li><a href="#"><img src="images/vector-1.png" alt="vector1"></a></li>
+                        <li><a href="#"><img src="images/vector-2.png" alt="vector2"></a></li>
+                        <li><a href="#"><img src="images/vector-3.png" alt="vector3"></a></li>
+                        <li><a href="#"><img src="images/vector-4.png" alt="vector4"></a></li>
+                        <li><a href="#"><img src="images/vector-5.png" alt="vector5"></a></li>
+                    </ul>
+                    <button>Get My Resume</button>
+                </div>
+            </div>
+        </div>
+        <div class="language-container">
+            <div class="languages">
+                <ul class="coder">
+                    <li class="coder-li"><a href="#">Languages</a><img class="coder-img" src="images/chevron.png" alt="languages"></li>
+                    <li>
+                        <ul class="lang-container">
+                            <li class="lang">
+                                <a href="#"><img class="lang-logo" src="images/javasript-logo.png" alt="javascript"></a>
+                                <p class="upper">Javascript</p>
+                            </li>
+                            <li class="lang">
+                                <a href="#"><img class="lang-logo" src="images/html-logo.png" alt="html"></a>
+                                <p class="upper">Html</p>
+                            </li>
+                            <li class="lang">
+                                <a href="#"><img class="lang-logo" src="images/css-logo.png" alt="css"></a>
+                                <p class="upper">css</p>
+                            </li>
+                        </ul>
+                    </li>
+                    <li class="coder-li"><a href="#">Frameworks</a><img class="coder-img" src="images/chevron.png" alt="frameworks"></li>
+                    <li class="coder-li"><a href="#">Skills</a><img class="coder-img" src="images/chevron.png" alt="skills"></li>
+                </ul>
+            </div>
+        </div>
+    </section>
```

Comment:
Languages, Skills, and Frameworks are not images. Please see the requirements again and make them a list.

Reply:
Thank you again. Yes, Languages Skills and Frameworks are not images. I already implemented that